src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -449,6 +449,15 @@
             az webapp create -g MyResourceGroup -p MyPlan -n MyUniqueAppName --runtime "node|6.2" --deployment-local-git
 """
 
+helps['webapp update'] = """
```

Comment:
Why do you have those changes? Please remove all changed files that are not related to MySQL/PG AllowAzureService